### PR TITLE
Update gir

### PR DESCRIPTION
**Pull request by ceyusa**
This change needs https://github.com/sdroege/gstreamer-sys/pull/18

### CODE REVIEW

File: gstreamer-audio/src/audio_format_info.rs [hunk 1]
```diff
@@ -126,7 +126,7 @@ impl AudioFormatInfo {
                 self.0,
                 flags.to_glib(),
                 dest.as_mut_ptr() as *mut _,
-                src.as_ptr() as *mut _,
+                src.as_ptr() as _,
```
Comment (sdroege) [inline gstreamer-audio/src/audio_format_info.rs]: Is a cast needed at all here now?

Comment (ceyusa) [reply gstreamer-audio/src/audio_format_info.rs]: afaiu yes, because u8 is not &[u8]

Comment (sdroege) [reply gstreamer-audio/src/audio_format_info.rs]: See https://github.com/sdroege/gstreamer-sys/pull/18/files#r226812757

**Comment by ceyusa** [comment]
fixed, remove the unneeded patch for audio and a quick fix for README.md

**Pull request merged**